src/components/map/MapService.js

```diff
@@ -1214,14 +1265,12 @@ goog.require('ga_urlutils_service');
         // or permalink
         // @param olLayerOrId  An ol layer or an id of a layer
         isExternalWmsLayer: function(olLayerOrId) {
-          if (!olLayerOrId) {
-            return false;
-          }
           if (angular.isString(olLayerOrId)) {
             return /^WMS\|\|/.test(olLayerOrId) &&
                 olLayerOrId.split('||').length >= 4;
+          } else {
+            return this.isWMSLayer(olLayerOrId);
```

Comment:
A priori, `isExternalWmsLayer` has nothing to do with tooltip.

To me the function name implies that the layer should an external integrated layer (via import tool or permalink). If I pass an internal layer as olLayer object to this function, it will return true, no?